src/Application.php

```diff
@@ -45,6 +45,10 @@ public function __construct($name = 'UNKNOWN', $version = 'UNKNOWN')
         $this->expressionParser = new ExpressionParser();
         $this->invoker = new Invoker();
 
+        if ($this->invoker->getParameterResolver() instanceof ResolverChain) {
```

Comment:
I wasn't sure how to go about this. Would you prefer to be more explicit and specify all the resolvers instead of relying on the default?